### PR TITLE
Update to Gradle 7.5

### PR DESCRIPTION
**Pull request by reta**
Signed-off-by: Andriy Redko <andriy.redko@aiven.io>

### Description
 Update to Gradle 7.5

### Issues Resolved
Closes https://github.com/opensearch-project/security/issues/1962

### Testing
N/A

### Check List
- [ ] New functionality includes testing
- [ ] New functionality has been documented
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 buildscript {
     ext {
-        opensearch_version = System.getProperty("opensearch.version", "2.1.0-SNAPSHOT")
```
Comment (peternied) [inline build.gradle:14]: https://github.com/opensearch-project/security/pull/1948 is already in flight to update to 2.2 - which should be merged shortly.  Once that is done lets rebase to update to the new gradle version

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-all.zip
```
Comment (peternied) [inline gradle/wrapper/gradle-wrapper.properties:3]: There is a second gradle-wrapper.properties at https://github.com/opensearch-project/security/blob/main/bwc-test/gradle/wrapper/gradle-wrapper.properties

**Review by peternied — CHANGES_REQUESTED** [review]

**Comment by peternied** [comment]
https://github.com/opensearch-project/security/pull/1948 has been merged, @reta could you rebase?  You should only need the changes to the gradle version

**Comment by reta** [comment]
> #1948 has been merged, @reta could you rebase? You should only need the changes to the gradle version

Yep, on it, thanks @peternied !

**Review by peternied — APPROVED** [review]
Thanks for the update!

**Review by cliu123 — APPROVED** [review]

**Pull request merged**